includes/class-wp-job-manager-post-types.php

```diff
@@ -659,6 +675,62 @@ public function job_feed() {
 		remove_filter( 'posts_search', 'get_job_listings_keyword_search' );
 	}
 
+	/**
+	 * Get filled jobs.
+	 *
+	 * @return array
```

Comment:
Might be helpful here to be explicit that this will return the IDs of the job listings.

Reply:
Makes sense, committed here https://github.com/Automattic/WP-Job-Manager/pull/2449/commits/ea63e6609cebe6de0d564deedd71abef6867bfc6